routes/user.js

```diff
@@ -24,7 +24,7 @@ sgMail.setApiKey(process.env.EMAIL_KEY);
 // User Login
 router.post(
   '/login',
-  auth,
+  //auth,
```

Comment:
We won't ever need login and signup to be protected routes, so you can just and the one below completely